kogito-kafka-quickstart-springboot/README.md

```diff
@@ -59,6 +110,13 @@ java -jar target/kogito-kafka-quickstart-sprintboot-{version}.jar
 
 To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* Send Message to Topic
+```
+kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
```

Comment:
As it is done for Kafka & Zookeeper starts, use `bin/kafka-console-producer.sh` in kafka folder